service/controller/internal/cloudconfig/template/etcd3_attach_dep_service.go

```diff
@@ -0,0 +1,29 @@
+package cloudconfig
+
+const Etcd3AttachDepService = `
+[Unit]
+Description=Attach etcd dependencies
+Requires=network.target
+After=network.target
+
+[Service]
+Environment="IMAGE={{ .RegistryDomain }}/giantswarm/aws-attach-etcd-dep:65a157aeb6db5cd4fa925efff792ef6fba4d02f3"
```

Comment:
Where is this image coming from? We should make it configurable maybe or at least have it in a common place where other images reside already.

Reply:
We have  images stored directly in the template for other services as well so just following common habbits.

Reply:
images goes from https://github.com/giantswarm/aws-attach-etcd-dep